packages/calcite-components/src/components/autocomplete-item-group/autocomplete-item-group.tsx

```diff
@@ -0,0 +1,75 @@
+import { LitElement, property, h, JsxNode } from "@arcgis/lumina";
+import { Scale } from "../interfaces";
+import { CSS } from "./resources";
+import { styles } from "./autocomplete-item-group.scss";
+
+declare global {
+  interface DeclareElements {
+    "calcite-autocomplete-item-group": AutocompleteItemGroup;
+  }
+}
+
+/** @slot - A slot for adding `calcite-autocomplete-item`s. */
+export class AutocompleteItemGroup extends LitElement {
+  // #region Static Members
+
+  static override styles = styles;
+
+  // #endregion
+
+  // #region Private Properties
+
+  // #endregion
+
+  // #region Public Properties
+
+  /**
+   * When `true`, signifies that the group should not have extra spacing. Used for styling.
+   *
+   * @private
+   */
+  @property() disableSpacing = false;
+
+  /** The component's text. */
+  @property() heading: string;
```

Comment:
Should this be marked `@required`?

Reply:
@geospatialem what do you think?

Reply:
Good eye by @anveshmekala! Agree since the `aria-label` falls back to the `heading` if a `label` isn't specified.